vic/drivers/image/src/vic_image_start.c

```diff
@@ -45,6 +45,17 @@ vic_image_start(void)
         get_global_param(filep.globalparam);
     }
 
+    if (mpi_rank == VIC_MPI_ROOT) {
+        // Open domain and parameter netCDF files
+        filenames.domain.nc_id = open_nc(filenames.domain.nc_file);
+        filenames.params.nc_id = open_nc(filenames.params.nc_file);
+    }
+
     // initialize image mode structures and settings
     vic_start();
+
+    if (mpi_rank == VIC_MPI_ROOT) {
+        // Close domain file
+        close_nc(filenames.domain);
+    }
```

Comment:
Unclear why we are doing this here rather than in `vic_start()`. Can we not do this inside the `VIC_MPI_ROOT` block in `vic_start`: https://github.com/UW-Hydro/VIC/blob/develop/vic/drivers/shared_image/src/vic_start.c#L69

I really want to avoid loading this information here if it is not needed at this level.